src/github/pullRequestGitHelper.ts

```diff
@@ -26,6 +27,10 @@ export interface PullRequestMetadata {
 export class PullRequestGitHelper {
 	static ID = 'PullRequestGitHelper';
 	static async checkoutFromFork(repository: Repository, pullRequest: PullRequestModel) {
+		if (!pullRequest.isResolved()) {
```

Comment:
In all of these cases where the PR has no head, we should display something to the user about why the action failed